app/src/main/java/dev/beefers/vendetta/manager/network/utils/CommitsPagingSource.kt

```diff
@@ -17,7 +19,7 @@ class CommitsPagingSource(
     override suspend fun load(params: LoadParams<Int>): LoadResult<Int, Commit> {
         val page = params.key ?: 0
 
-        return when (val response = repo.getCommits("Vendetta", page)) {
+        return when (val response = repo.getCommits(prefs.vendettaModuleRepo, page)) {
```

Comment:
Now displaying commits from Xposed module

Reply:
The commits of the module aren't what's relevant, the user doesn't care how the app is patched, they care about the mod injected afterwards as that is what they're using